src/winforms/toga_winforms/widgets/multilinetextinput.py

```diff
@@ -12,7 +11,7 @@ def create(self):
         self.native.Multiline = True
 
     def set_readonly(self, value):
-        self.native.ReadOnly = value
+        self.native.ReadOnly = self.interface.readonly
```

Comment:
If the value is not going to be used, does it need to still be a parameter?

Reply:
I think the value passed is not cleaned, so we use the interface 'cleaned' value instead, as per #299